crates/store/re_types/definitions/rerun/blueprint/archetypes/map_options.fbs

```diff
@@ -0,0 +1,21 @@
+namespace rerun.blueprint.archetypes;
+
+
+/// Configuration for the background of a view.
+table MapOptions (
+    "attr.rerun.scope": "blueprint"
+) {
+    // --- Required ---
+
+    /// Map provider and style to use.
+    provider: rerun.blueprint.components.MapProvider ("attr.rerun.component_required", order: 1000);
+
+
+    // --- Optional ---
+
+    /// Zoom level for the map. The default is 16.
+    zoom: rerun.blueprint.components.ZoomLevel ("attr.rerun.component_optional", order: 2000);
```

Comment:
In addition to zoom level, it would be extremely valuable to store in the blueprint the scroll position (coordinate of center point?). In particular, that would make it settable from code.

If that's too involved, we can keep it for a follow-up PR.

Reply:
absolutely possible, but the map center can be more than a position:

 - it can be in follow mode, which is useful if you have one or interconnected points. here the first logged point is always in the center (like in the embedded video on the PR description)
 - we can fix the map at a certain lat, long position

what archetypes should I use to store this? in the lib if the map center is `None` then it is in the following mode; otherwise, it is fixed. but I assume this approach is not idiomatic in rerun

Reply:
According to your last comment, the `center_position` should be `DVec2D`, and `follow_mode` should be a new enum or a boolean with a well-named entry to keep the number of components down.

Reply:
My gut feeling would be to minimise the feature set for this PR and skip any sort of "auto-scroll" mode. The MVP would be:
- `CenterPosition` view property (datatype: `DVec2D`) to store the center location along with the zoom level
- Best effort implementation for the fallback provider of `CenterPosition` based on whatever data is logged

In the future, we indeed need to devise a set of more advanced auto-scroll/follow mode, but I think the hindsight we'll gain from landing this PR will help designing that properly. (Ideally, it'd be somewhat consistent with the 2D view too.)